src/transform/tests/test_runner.rs

```diff
@@ -180,7 +183,38 @@ mod tests {
                 json_to_spec(&serde_json::to_string(rel).unwrap(), cat).0
             ),
             FormatType::Json => format!("{}\n", serde_json::to_string(rel).unwrap()),
-            FormatType::Explain(format) => generate_explanation(cat, rel, *format),
+            FormatType::Explain(format) => {
+                let format_contains = |key: &str| {
+                    format
+                        .map(|format| format.contains(&key.to_string()))
+                        .unwrap_or(false)
+                };
+
+                let config = ExplainConfig {
+                    arity: false,
+                    join_impls: false,
```

Comment:
Would it be ok to make this `true`? E.g., `src/transform/tests/testdata/join-implementation` needs this info. Note that it's also `true` in `pretty`.

Reply:
Yes, I copy-pasted the `pretty()` config, sorry.